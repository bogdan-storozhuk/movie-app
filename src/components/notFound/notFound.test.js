import React from "react";
import { render } from "@testing-library/react";

import NotFound from "./";

describe("NotFound", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<NotFound>Example</NotFound>);

    expect(asFragment(<NotFound>Example</NotFound>)).toMatchSnapshot();
  });
});
