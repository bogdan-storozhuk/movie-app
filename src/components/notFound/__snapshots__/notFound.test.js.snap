// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should match snapshot 1`] = `
<DocumentFragment>
  <h1
    class="NotFound"
  >
    Not Found
  </h1>
</DocumentFragment>
`;
